README.md

```diff
@@ -13,11 +13,9 @@ Requirements
 This role requires:
 
 - Ansible version > 2.0
-- Beats versions 1.x
+- Beats version > 1.x
```

Comment:
Should be `Beats version >= 1.x`. Though I hope no one is still running beats 1.X :D